pkgs/applications/audio/VoiceOfFaust/default.nix

```diff
@@ -0,0 +1,63 @@
+{ stdenv, pkgs, callPackage, fetchFromGitHub, faust2jack, helmholtz, mrpeach }:
+stdenv.mkDerivation rec {
+  name = "VoiceOfFaust-${version}";
+  version = "v0.7";
+
+  src = fetchFromGitHub {
+    owner = "magnetophon";
+    repo = "VoiceOfFaust";
+    rev = "${version}";
+    sha256 = "14jjs7cnhg20pzijgblr7caspcpx8p8lpkbvjzc656s9lqn6m9sn";
+  };
+
+  plugins = [ helmholtz mrpeach ];
+
+  pitchTracker = callPackage ../puredata/wrapper.nix { inherit plugins; };
```

Comment:
@pSub Are you saying I should add it to all-packages.nix?  Or do you mean this somehow adds it?